src/Pagination/Connection.hack

```diff
@@ -50,13 +50,15 @@ type PaginationArgs = shape(
  * @see https://relay.dev/graphql/connections.htm for more information about GraphQL pagination.
  * @see src/playground/UserConnection.hack for an example.
  */
+ <<GraphQL\ObjectType('Connection', 'Connection')>>
```

Comment:
I actually realize this isn't necessary. We don't generate a GQL object for `Connection` so we don't need to tag the class with this attribute.